services/symfony/symfony-insight-grade.tester.js

```diff
@@ -37,114 +19,10 @@ createTest(t, 'live: valid project grade', { withMockCreds: false })
     ).required(),
   })
 
-createTest(t, 'live: nonexistent project', { withMockCreds: false })
-  .before(prepLiveTest)
+t.create('live: nonexistent project')
```

Comment:
🤔 Should we remove the `live:` prefix from these tests in light of #3661? These conditional skip tests are a little tricky since sometimes their live but others mocked.

Reply:
Now they’re always live, right? Live, or skipped. But yea, I could update these titles.

Reply:
> Now they’re always live, right? Live, or skipped

Ah yes you are correct. I didn't pay close enough attention the change